pymc3/stats.py

```diff
@@ -795,6 +782,10 @@ def df_summary(trace, varnames=None, transform=lambda x: x, stat_funcs=None,
         mu__0  0.106897  0.066473  0.001818 -0.020612  0.231626
         mu__1 -0.046597  0.067513  0.002048 -0.174753  0.081924
 
+                  n_eff      Rhat
+        mu__0       6.0  7.509734
```

Comment:
here you should change the n_eff and Rhat to a more reasonable number, eg n_eff=487.0 Rhat=1.00001